deploy/eck-operator/templates/role-bindings.yaml

```diff
@@ -0,0 +1,80 @@
+{{- $operatorNSIsManaged := has .Release.Namespace .Values.managedNamespaces -}}
+{{- $fullName := include "eck-operator.fullname" . -}}
+{{- $svcAccount := include "eck-operator.serviceAccountName" . }}
+
+{{- if not .Values.createClusterScopedResources }}
+{{- range .Values.managedNamespaces }}
+{{- $namespace := . }}
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: "{{ $fullName }}"
+  namespace: {{ $namespace }}
+  labels:
+    {{- include "eck-operator.labels" $ | nindent 4 }}
+rules:
+{{ template "eck-operator.rbacRules" $ | toYaml | indent 2 }}
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: "{{ $fullName }}"
+  namespace: {{ $namespace }}
+  labels:
+    {{- include "eck-operator.labels" $ | nindent 4 }}
```

Comment:
I think the Selector labels include is missing a `-` as it produces a leading empty line.